tests/js/spec/components/charts/optionSelector.spec.tsx

```diff
@@ -105,7 +105,7 @@ describe('EventsV2 > OptionSelector (Multiple)', function () {
     expect(onChangeStub).toHaveBeenCalledWith([
       'count()',
       'failure_count()',
-      'avg(transaction.duration)',
+      'count_unique(user)',
```

Comment:
Not sure how this test passed before, but the third dropdown item that we've just selected (see the line immediately above this) is `count_unique(user)`, not `avg(transaction.duration)`.